src/fastlane/release_actions/lib/fastlane/plugin/release_actions/version.rb

```diff
@@ -1,5 +1,5 @@
 module Fastlane
   module ReleaseActions
-    VERSION = "0.1.0"
+    VERSION = "1.0.0"
```

Comment:
This is version 1.0.0 of the Fastlane release actions? Or version 1.0.0 for the thing(s) it will release?

Reply:
Yessir! Of the fastlane release_actions plugin